casbin/persist/adapter.py

```diff
@@ -32,7 +32,10 @@ def load_policy_line(line, model):
     if key not in model.model[sec].keys():
         return
 
-    model.model[sec][key].policy.append(tokens[1:])
+    if model.has_policy(sec, key, tokens[1:]):
```

Comment:
not necessary, because `model.has_policy` gets called inside `model.add_policy`.

Reply:
Thanks for reminding! I have removed the redundant ```has_policy```, could you please review it again?